Sources/Tracking/CustomerIOImplementation.swift

```diff
@@ -91,9 +99,11 @@ public class CustomerIOImplementation: CustomerIOInstance {
         // don't modify the state of the SDK until we confirm we added a background queue task successfully.
         // XXX: better handle scenario when adding task to queue is not successful
         if queueStatus.success {
```

Comment:
```suggestion
        if !queueStatus.success {
            // XXX: better handle scenario when adding task to queue is not successful
            logger.debug("failed to enqueue identify task")
            return
        }
```

Reply:
would reduce nesting of the happy path below, and make the comment a little clearer